### PR TITLE
feat: use new build script and kmc

### PR DESCRIPTION
**Pull request by mcdurdin**
Replaces the old build script with a cleaner version that utilizes builder.inc.sh, standardizing our build commands.

Uses `kmc` to build all source models, in a single command rather than one-by-one as previously. Handles external models more simply.

TODO: 
- [x] Awaiting merge of changes to kmc to keymanapp/keyman/master before committing updated package.json; this will not build until the updated kmc is available. (See keymanapp/keyman#9717)
- [x] ` .../release/packages/fv_all/../../../resources/builder.inc.sh: line 1079: KEYMAN_ROOT: unbound variable` (See 3e1368dd649631ab1b4e91d5c41b286ef949e669)
- [x] Update LICENSE.md (See #2404)

---

Please note: this will merge into staging-17.0, in preparation for 17.0 release. Changes on the staging-17.0 branch will not be published to keyman.com, until they are merged into master when we go to release (or when we determine the updated build tools are sufficiently stable).

Please ignore the failing build in this PR, which is due to data clean required and is addressed in follow-up PRs in the chain (particularly #2404).

Other PRs in this chain:
* #2404 
* #2406 
* #2407 
* #2408 
* #2409 
* #2410 
* #2413 
* #2414 
* #2415
* #2419

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ multiple keyboards share the same prefix indicating provenance, for example `sil
 
 First, read the guide! <https://help.keyman.com/developer/keyboards/>
 
-Keyman Developer 12.0 is recommended to edit these files. <https://keyman.com/developer>
+Keyman Developer 17.0+ is recommended to edit these files. <https://keyman.com/developer>
```
Comment (darcywong00) [inline README.md:20]: The build script with kmc can run without WINE now, right? (ll. 26-38)

Comment (mcdurdin) [reply README.md:20]: Yes, correct. I need a separate PR I think to address this.

File: release/packages/fv_all/build.sh [hunk 2]
```diff
@@ -3,186 +3,163 @@
...
-    OSK_FONT_LINES_0='
-      <OSKFont>'"$oskFont"'</OSKFont>'
-  fi
-  DISPLAY_FONT_LINES_0=''
-  if [[ "$displayFont" != 'none' ]]; then
-    DISPLAY_FONT_LINES_0='
-      <DisplayFont>'"$displayFont"'</DisplayFont>'
+function do_clean() {
+  rm -f ./source/fv_all.kps
+  rm -rf ./build/
```
Comment (darcywong00) [inline release/packages/fv_all/build.sh:33]: Do you want to clean these files too?
```
rm -f build_targets.txt build_targets_temp.txt build_external_targets.txt build_fv_all.txt build_legacy.txt
```

Comment (darcywong00) [reply release/packages/fv_all/build.sh:33]: ah nvm - I meant in the top build.sh do_clean

File: release/packages/fv_all/build.sh [hunk 3]
```diff
@@ -3,186 +3,163 @@
...
-done
+  kps=$(<source/fv_all.kps.in)
+  kps="${kps//@FILES/$FILE_LINES}"
+  echo "${kps//@KEYBOARDS/$KEYBOARD_LINES}" > source/fv_all.kps
 
-# Verify keyboards exist
-if [[ $KEYBOARD_LINES = '' ]]; then
-  die "No built keyboards exist for fv_all"
-fi
+  # Build fv_all.kmp, using kmcomp as per normal build script
```
Comment (darcywong00) [inline release/packages/fv_all/build.sh:147]: This is now kmc?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (mcdurdin) [reply README.md:20]: Addressed in #2419

File: release/packages/fv_all/build.sh (hunk 2 above)
Comment (mcdurdin) [reply release/packages/fv_all/build.sh:33]: gotcha

**Pull request merged**